api/src/main/java/org/openmrs/ProgramWorkflow.java

```diff
@@ -17,29 +17,52 @@
 import java.util.Set;
 import java.util.TreeSet;
 
+import javax.persistence.CascadeType;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.persistence.OneToMany;
+import javax.persistence.OrderBy;
+import javax.persistence.Table;
+
 import org.hibernate.envers.Audited;
 import org.openmrs.util.NaturalStrings;
 
 /**
  * ProgramWorkflow
  */
 @Audited
+@Entity
+@Table(name = "program_workflow")
 public class ProgramWorkflow extends BaseChangeableOpenmrsMetadata {
 	
 	private static final long serialVersionUID = 1L;
 	
 	// ******************
 	// Properties
-	// ******************
-	
+	// *****************
+	@Id
+    @GeneratedValue(strategy = GenerationType.SEQUENCE, generator = "program_workflow_program_workflow_id_seq")
```

Comment:
Collect the identification here, Did you run the jar file to comfirm all is working. ?